src/lsqecc/pauli_rotations/rotation.py

```diff
@@ -78,10 +78,10 @@ def multiply_operators(a: "PauliOperator", b: "PauliOperator"):
 ] = {
     (PauliOperator.Z, PauliOperator.X): (1j, PauliOperator.Y),
     (PauliOperator.X, PauliOperator.Z): (-1j, PauliOperator.Y),
-    (PauliOperator.Z, PauliOperator.Y): (1j, PauliOperator.X),
-    (PauliOperator.Y, PauliOperator.Z): (-1j, PauliOperator.X),
-    (PauliOperator.Y, PauliOperator.X): (1j, PauliOperator.Z),
-    (PauliOperator.X, PauliOperator.Y): (-1j, PauliOperator.Z),
+    (PauliOperator.Y, PauliOperator.Z): (1j, PauliOperator.X),
```

Comment:
how did I even miss this… 😱

Reply:
I think there was some confusion about what this table is, we need to comment with what is the invariant between the tuple elements:
`(A,B),(k,C) => A*B = k*C`